source/arm9/libc/chdir.c

```diff
@@ -101,7 +101,7 @@ char *getcwd(char *buf, size_t size)
             optimize_mem = 1;
         }
 
-        buf = calloc(size, 1);
+        buf = malloc(size);
```

Comment:
This can just be a `malloc`, as I think we can trust `getcwd` to return a valid string if no error has been raised?

Reply:
Sounds good.